### DISCUSSION TITLE
孤儿 Forge 作者

### DISCUSSION THREAD

**Discussion by hejiehao** [Meloong-Git/PCL#3785] [讨论]
https://github.com/Hex-Dragon/PCL2/blob/d682b598032bc4c7e3b1367cae4d51d037007ae9/Plain%20Craft%20Launcher%202/Modules/Minecraft/ModMinecraft.vb#L1635

**Comment by Deep-Dark-Forest** [comment]
forge作者其实不是孤儿（）

**Comment by hejiehao** [comment]
如果我发到 Forge 的 Discord 服务器会发生什么？

**Reply by Ad-closeNN** [reply]
有可能会被禁言（应该是先警告）

**Reply by iPodtouch6557** [reply]
绝知此事要躬行（）

**Reply by hejiehao** [reply]
https://discord.com/channels/1129059589325852724/1129075001300107324/1236310247656788011

**Reply by Ad-closeNN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/6316a485-1542-46f5-8edc-3241a86f61d1)
@hejiehao 可否给予dc服务器邀请链接？

**Reply by hejiehao** [reply]
https://discord.minecraftforge.net/
